src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishASelfContainedApp.cs

```diff
@@ -129,7 +111,9 @@ public void It_can_make_a_Windows_GUI_exe()
                 .Be(2);
         }
 
-        [Fact]
+        // TODO: This test should be fixed by the fixes in MSBuild here:
+        // Won't merge this code in until that has flown in, elsewise libraries which are rid agnostic will get rids.
+        [Fact(Skip = "https://github.com/dotnet/msbuild/pull/6924")]
```

Comment:
Code from another PR linked above needs to be merged in for this to work because SelfContained is being pushed into library code that isn't actually self-contained. This change breaks that until that other change is merged in and fixes the issue.